source/dlangbot/app.d

```diff
@@ -372,9 +372,13 @@ void handlePR(string action, PullRequest* _pr)
             );
 
             updateBugs([r.id], issueComment, r.fixed);
+            auto statuses = descs.filter!(d => d.id == r.id);
+            auto status = statuses.empty ? null : statuses.front.status;
```

Comment:
I think you need to filter `descs` itself. Below, it's still adding the entire `descs` regardless of their `status`.